src/libraries/System.Private.CoreLib/src/System/IO/Strategies/WindowsFileStreamStrategy.cs

```diff
@@ -28,6 +27,7 @@ internal abstract class WindowsFileStreamStrategy : FileStreamStrategy
         protected long _filePosition;
         private long _appendStart; // When appending, prevent overwriting file.
         private long _length = -1; // When the file is locked for writes (_share <= FileShare.Read) cache file length in-memory, negative means that hasn't been fetched.
+        private bool _exposed; // created from handle, or SafeFileHandle was used and the handle got exposed
```

Comment:
nit: this is the name used in Legacy.
```suggestion
        private bool _exposedHandle; // created from handle, or SafeFileHandle was used and the handle got exposed
```